src/components/formCovid/index.tsx

```diff
@@ -1,25 +1,88 @@
 import { FOAF } from '@inrupt/lit-generated-vocab-common';
 import { Text, useSession } from '@inrupt/solid-ui-react';
 import { useObserver } from 'mobx-react-lite';
+import pdfjs from 'pdfjs-dist';
 import React, { FC, useState } from 'react';
 
 import { useStores } from '../../contexts/index';
+import CovidInformation from '../covidInformation';
+
+pdfjs.GlobalWorkerOptions.workerSrc =
+  'https://cdn.bootcss.com/pdf.js/2.4.456/pdf.worker.js';
 
 const FormCovid = (): FC => {
   const { session } = useSession();
   const [certificaat, setCertificaat] = useState('vaccinatiecertificaat');
-  const [date, setDate] = useState();
   const [file, setFile] = useState();
+  const [date, setDate] = useState<string>('');
+  const [dosis, setDosis] = useState<string>('');
+  const [id, setId] = useState<string>('');
+  const [group, setGroup] = useState<string>('wheelhouse');
   const { solidStore } = useStores();
 
-  const handleFiles = (e: any): void => {
-    const file = e.target.files[0];
-    setFile(file);
+  const handleDate = (e: any): void => {
```

Comment:
Ik heb bij onChange handlers altijd de gewoonte om achteraan nog Change te schrijven. Als je dan de functie ziet dan weet je dat deze op een onChange reageert. In dit geval `handleDateChange`